config/environments/production.rb

```diff
@@ -75,4 +75,16 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.delivery_method = :smtp
+  
+  config.action_mailer.smtp_settings = {
+      address: "smtp.sendgrid.net",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.